shared/attributes.asciidoc

```diff
@@ -536,12 +538,6 @@ Legacy definitions
 :apm-overview-ref-70:  https://www.elastic.co/guide/en/apm/get-started/7.0
 :apm-overview-ref-m:   https://www.elastic.co/guide/en/apm/get-started/master
 :infra-guide:          https://www.elastic.co/guide/en/infrastructure/guide/{branch}
-// Use data-sources-caps instead of Data-Sources
-:Data-Sources:         Data Views
-// Use data-source-cap instead of Data-source
-:Data-source:          Data view
-// Use data-sources-cap instead of Data-sources
-:Data-sources:         Data views
 // Do not use A-data-source or a-data-source
 :A-data-source:      A data view
 :a-data-source:      a data view
```

Comment:
Should we also remove these case-sensitive attributes, or cut one of them to avoid duplicates? I don't know how widely they're used, or if they're causing problems like the other ones were.

Reply:
"a-data-source" is still used in some versions (e.g. 8.11 and earlier) so I think you're right that we can remove the "A-data-source" but keep the other.

Reply:
Fixed in b42131d62d2e8ec00e690170d1132398539dfea6